### PR TITLE
Use line height equal to icon height

### PR DESCRIPTION
**Pull request by testbrian**
screen shot 2016-01-21 at 21 55 03

### CODE REVIEW

**Comment by testbrian** [comment]
Wow it's great to see this feature being used right now. If you check http://rubyjobs.ie/ you will see there is a slight line height issue.

**Comment by vyncem** [comment]
Yea it looks beautiful alright. :-)

I see what you mean about the line height, it's currently at 20px and 
looks to be good at 15px.

If you'd like to do a PR I'll can push it through.

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -199,6 +199,7 @@ body {
 	.job {
 		border-bottom: 1px solid #ccc;
 		padding: 6px;
+    line-height: 20px;
```
Comment (vyncem) [inline app/assets/stylesheets/application.css]: I'd change #L225 to height 15px other than making all the lines bigger.

Why do you prefer this solution?

|.job.remote_icon img - 15px| current broken state |
|---|---|
|screen shot 2016-04-29 at 12 37 30|screen shot 2016-04-29 at 12 36 46|


https://github.com/rubyireland/rubyjobs2/pull/16/files#diff-fb7484bc2f56c263954da6fc44982eeaR226

Comment (testbrian) [inline app/assets/stylesheets/application.css]: I definitely wouldn't go with a smaller icon, it's difficult enough to see what it is as is.

Comment (vyncem) [inline app/assets/stylesheets/application.css]: fair enough. :-)

**Pull request merged**